interoptest/src/pythonservice/service.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=no-member
+# pylint: disable=invalid-name
+# pylint: disable=missing-docstring
+# pylint: disable=too-few-public-methods
+
+from uuid import uuid4
+import logging
+import sys
+
+from opencensus.trace.ext import requests as requests_wrapper
+from opencensus.trace.tracers import context_tracer
+import grpc
+import requests
+
+import interoperability_test_pb2 as pb2
+import interoperability_test_pb2_grpc as pb2_grpc
+
+# Monkey patch the requests library
+requests_wrapper.trace.trace_integration(context_tracer.ContextTracer())
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+REGISTRATION_SERVER_HOST = ''
+REGISTRATION_SERVER_PORT = ''
+
+GRPC_TPE_WORKERS = 10
+HTTP_POST_PATH = "/test/request/"
+
+PORT_MAP = {
```

Comment:
PORT_MAP is used for validation only. If new service (let say Ruby)  is added then you will have to update this. Not having this would result into request timeout if wrong port/transport is sent in the request. Either way it will fail. One will fail right away and other will fail after a timeout. I would recommend to remove it.